features/step_definitions/timehelpers_steps.rb

```diff
@@ -1,20 +1,31 @@
+# frozen_string_literal: true
+
+After do
+  travel_back
+end
+
+def no_nest_travel_to(time, &block)
```

Comment:
I'm wondering - if we add a Before hook with `travel_back`, maybe we can use just `travel_to`, with no extra method?

Otherwise, I think a comment here would be useful, I mean explaining why this method is needed, which is I assuming because TimeHelpers does not two consequent `travel_to`, so `travel_back` is required before it.

Reply:
you mean in after hook?

Reply:
both After and Before :)

Reply:
As you guessed, the reason for that method is to avoid nested time travel, which is not recommended in TimeHelpers. I added the tests pass, so I'm committing it.

Reply:
Some tests failed in CircleCI, so I implemented the method again, with some changed